bucket-policy.go

```diff
@@ -336,9 +337,9 @@ func unMarshalBucketPolicy(bucketPolicyBuf []byte) (BucketAccessPolicy, error) {
 	var policyUntyped = bucketAccessPolicyUntyped{}
 	// Unmarshal incoming policy into an untyped structure, to be
 	// evaluated lazily later.
-	err := json.Unmarshal(bucketPolicyBuf, &policyUntyped)
-	if err != nil {
-		return BucketAccessPolicy{}, err
+	e := json.Unmarshal(bucketPolicyBuf, &policyUntyped)
```

Comment:
Don't use 'e' here since here we don't need differentiation between probe.Error and regular golang error.